mplc/partner.py

```diff
@@ -103,4 +103,5 @@ def split_minibatches(self):
         self.minibatched_y_train = np.split(y_train, (split_indices[:-1] * len(y_train)).astype(int))
 
     def build_model(self):
+        # TODO Why don't we have the model as a parameter of partnerMPL object ?
```

Comment:
Because for the keras models we store only the weights and discard the tensorflow graphe to save memory. So we need to rebuild it. 
It is the role of the clear_session() method